server/models/Message.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, DataTypes) => {
+  const Messages = sequelize.define('Messages', {
+    senderId: DataTypes.INTEGER,
+    receiverId: DataTypes.INTEGER,
+    parentMessageId: DataTypes.INTEGER,
+    body: DataTypes.STRING
```

Comment:
All this should have some constraints, they shouldn't be nullable especially for the `senderId` `receiverId` and body.

Reply:
Okay @fortunee